src/App.js

```diff
@@ -9,23 +9,26 @@ import './index.css';
 export default class App extends Component {
 	constructor(props) {
 		super(props);
+		
 		this.state = {isDrawerOpened: false};  
 	}
 
 	render() {
-		const drawer = this.state.isDrawerOpened 
-			? <Drawer onClick={this.toggleDrawer}/> 
-			: null;
-
 		return (
 			<div className="layout">
-				{drawer}
+				{this.getDrawer()}
```

Comment:
If some method returns jsx markup, it is better to name them like `render[ElementName]`.